bindings/c/test/apitester/TesterTransactionExecutor.cpp

```diff
@@ -81,6 +81,8 @@ class TransactionContextBase : public ITransactionContext {
 	  : fdbTx(tx), txActor(txActor), contAfterDone(cont), scheduler(scheduler), retryLimit(retryLimit),
```

Comment:
We could, but I prefer maintaining a strict order cleanup:
- All workloads complete
- join with the scheduler: All pending scheduler tasks complete
- destroy scheduler
- join with the network thread: All pending tasks and callbacks of the network thread complete
- destroy FDB network state

So we have a clearly defined lifetime of the scheduler, and no need to make it more complicated by reference counting.

When all workloads are declared as completed, all remaining TransactionContexts should be in DONE state. No new tasks should be scheduled. The TransactionContexts are kept only because they are passed as parameters to the callbacks.